src/commands/CreateVDBCommand.ts

```diff
@@ -14,35 +14,35 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import * as vscode from 'vscode';
-import * as path from 'path';
+import * as constants from '../constants';
+import * as extension from '../extension';
 import * as fs from 'fs';
+import * as path from 'path';
 import * as utils from '../utils';
-import * as extension from '../extension';
+import * as vscode from 'vscode';
+import { DataVirtConfig } from '../model/DataVirtModel';
 import { SchemaTreeNode } from '../model/tree/SchemaTreeNode';
-import { IDVConfig } from '../model/DataVirtModel';
 
 export function createVDBCommand() {
 	if (extension.workspaceReady) {
-		vscode.window.showInputBox( {validateInput: (name: string) => {
+		vscode.window.showInputBox( { validateInput: (name: string) => {
 			let res: string = utils.validateName(name);
 			if (!res) {
-				// check if file already exists
 				res = utils.validateFileNotExisting(name);
 			}
 			return res;
-		}, placeHolder: 'Enter the name of the new VDB config'})
-			.then( (fileName: string) => {
-				handleVDBCreation(vscode.workspace.workspaceFolders[0].uri.fsPath, fileName)
+		}, placeHolder: 'Enter the name of the new VDB config' })
+			.then( (vdbName: string) => {
+				if (!vdbName) {
+					return;
+				}
+				handleVDBCreation(vscode.workspace.workspaceFolders[0].uri.fsPath, vdbName)
 					.then( (success: boolean) => {
 						if (success) {
-							const node: SchemaTreeNode = extension.dataVirtProvider.getSchemaTreeNodeOfProject(fileName);
-							if (node) {
-								vscode.commands.executeCommand('datavirt.edit.schema', node);
-							}
-							vscode.window.showInformationMessage(`New VDB ${fileName} has been created successfully...`);
+							openDDLEditor(vdbName);
```

Comment:
Are we guaranteed that if the VDB is created successfully that the DDL editor will open successfully? Is there a chance it could fail? If so, should we have some kind of error checking in/around this to ensure that it doesn't fail silently?

Reply:
If the command itself returns true then we have the vdb file created successfully. That said the open should work.